streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/tasks/InputStreamTask.java

```diff
@@ -0,0 +1,49 @@
+package org.ray.streaming.runtime.worker.tasks;
+
+import org.ray.runtime.util.Serializer;
+import org.ray.streaming.runtime.core.processor.Processor;
+import org.ray.streaming.runtime.transfer.Message;
+import org.ray.streaming.runtime.worker.JobWorker;
+
+public abstract class InputStreamTask extends StreamTask {
+  private volatile boolean running = true;
+  private volatile boolean stopped = false;
+
+  public InputStreamTask(int taskId, Processor processor, JobWorker streamWorker) {
+    super(taskId, processor, streamWorker);
+  }
+
+  @Override
+  protected void init() {
+  }
+
+  @Override
+  public void run() {
+    while (running) {
+      Message item = reader.pull(10);
```

Comment:
reader.pull(timeout);
this parameter should be defined as a field in configuration.

Reply:
fixed